core/src/test/java/org/elasticsearch/index/seqno/LocalCheckpointTrackerTests.java

```diff
@@ -49,16 +49,7 @@
     private static final int SMALL_CHUNK_SIZE = 4;
 
     public static LocalCheckpointTracker createEmptyTracker() {
-        return new LocalCheckpointTracker(
-            IndexSettingsModule.newIndexSettings(
-                "test",
-                Settings
-                    .builder()
-                    .put(LocalCheckpointTracker.SETTINGS_BIT_ARRAYS_SIZE.getKey(), SMALL_CHUNK_SIZE)
-                    .build()),
-            SequenceNumbers.NO_OPS_PERFORMED,
-            SequenceNumbers.NO_OPS_PERFORMED
-        );
+        return new LocalCheckpointTracker(SequenceNumbers.NO_OPS_PERFORMED, SequenceNumbers.NO_OPS_PERFORMED);
```

Comment:
Wait - I think we need to allow testing it with a small value - to make sure multiple arrays are used? I was thinking of adding a (protected/package private/whatever) constructor.

Reply:
I think we had a miscommunication, which we have now clarified via another channel. I have pushed so that we test with a lot of operations to cover the default size of the bit sets we are using now, as opposed to the small size that we were using before.